Gemfile

```diff
@@ -12,6 +12,7 @@ gem "activestorage", RAILS_VERSION
 gem "activesupport", RAILS_VERSION
 gem "railties", RAILS_VERSION
 
+gem "cssbundling-rails"
```

Comment:
Should we use the normal alphabetical ordering here?

Reply:
I thought it might be nice to have the asset-related gems as a separate section, but we can put them in the Big Long List™ too – not fussed!